### PR TITLE
Ifpair

### PR DESCRIPTION
**Pull request by SophieRoseMcDonald**
I have updated the image file names to reflect they are from team ifpairelseunknown. I have also update the link to Adam's twitter account, previously it was not working. Lastly, I deleted an extra file which was created and not required.

### CODE REVIEW

File: blog/_posts/2017-07-17-ifPairElseUnknown.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+current: blog
+author: ifPairElseUnknown
+twitter: IfPairElse  ‏
+categories:
+- student-posts
+---
+
+# __Hello World!__
+
+![Team Banner](/img/blog/2017/ifPairElseUnknown_v5.png)
```
Comment (alicetragedy) [inline blog/_posts/2017-07-17-ifPairElseUnknown.md:15]: please add image credits to all the images you've used in the post. “Image credits” means either a source for the images/where you got them from, or who took the photo — depending on what type of image it is. you can find more about this [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images). Please also take a look at past blog posts to see the styling that's been applied to the credits :)

File: Gemfile.lock [hunk 2]
```diff
@@ -1,26 +1,25 @@
 GEM
```
Comment (alicetragedy) [inline Gemfile.lock]: a little issue here — `Gemfile.lock` has been committed, which it shouldn't have. (i.e. for now we want to stick to the previous version of the `Gemfile.lock` file.) To undo this, you can: 
- rollback to the previous version of the `Gemfile.lock`
- create a new commit with that new change (i.e. the new change being the switch from your “new” `Gemfile.lock` to our old `Gemfile.lock`)
- push the new change to your branch (this should update the PR accordingly)

I hope this helps. If it doesn't, or if it seems too cryptic, please ask your coaches or supervisor for help, or ping me and I can try to help! :)

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hi @SophieRoseMcDonald and @jnamster, thanks for your PR! ✨  Before we can merge, there are a couple of small things that need to be changed:
- adding image credits
- removing the changes made to the `Gemfile.lock` file
I've commented on these two things more in detail as inline comments on the files. 
Please update your PR accordingly when you can, and we'll be ready to publish and share your post after that!

**Comment by SophieRoseMcDonald** [comment]
I have made those changes. Please let me know if there is anything else that needs to be done.

**Review by alicetragedy — APPROVED** [review]
looks great, thank you for making those changes @SophieRoseMcDonald! And congratulations on your first contribution to the `summer-of-code` repo! ✨

**Pull request merged**